.planning/codebase/STRUCTURE.md

```
@@ -0,0 +1,227 @@
+# Directory Structure
+
+## Root (`/workspaces/KMP/`)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)